app/auth-portal/src/components/MemberListItem.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <tr
+    class="children:px-md children:py-sm children:truncate text-sm font-medium text-gray-800 dark:text-gray-200"
+  >
+    <td class="">
+      <div
+        class="xl:max-w-[800px] lg:max-w-[60vw] md:max-w-[50vw] sm:max-w-[40vw] max-w-[150px] truncate"
+      >
+        {{ memUser.email }}
+      </div>
+    </td>
+    <td class="normal-case">
+      <ErrorMessage :requestStatus="changeMembershipReq" />
+      <VormInput
+        v-model="changeWorkspaceRoleType"
+        :options="changeWorkspaceRoleTypeDropdownOptions"
+        :disabled="
+          !featureFlagsStore.CHANGE_USER_ROLE ||
+          memUser.role === 'OWNER' ||
+          !isWorkspaceOwner
+        "
+        label="Role"
```

Comment:
```suggestion
        noLabel
```

Since this is an input in a named column, it'd be better to just hide the label on the form field itself